src/dune_rules/slang_expand.mli

```diff
@@ -11,12 +11,12 @@ type expander =
     lazily. Undefined package variables or invalid strings in the conditions of
     if and when statements do not result in errors if they occur in unevaluated
     portions of an expression. *)
-val eval : Slang.t -> dir:Path.t -> f:expander -> Value.t list Memo.t
+val eval : Slang.t -> dir:Path.t -> f:expander -> Value.t list list Memo.t
```

Comment:
Actually, this function wasn't even used outside of this module. I made it private. Also added a  constructors so that callers are clear that we are just concatenating the values.